src/test/resources/n.json

```diff
@@ -61,6 +61,6 @@
     "num2" : 3,
     "r" : 2.0999999999999998E-4,
     "x" : 0.017998774958313132,
-    "r1" : 1.0013337777777775
+    "r1" : 1.0013337777777778
```

Comment:
why is it different?

Reply:
Because I have introduced `baseRatio` as an intermediate.